inginious/frontend/tasks.py

```diff
@@ -17,7 +17,7 @@
 class WebAppTask(Task):
     """ A task that stores additional context information, specific to the web app """
 
-    def __init__(self, course, taskid, content, task_fs, translations_fs, hook_manager, task_problem_types):
+    def __init__(self, course, taskid, content, task_fs, translations_fs, hook_manager, task_problem_types, debug=False):
```

Comment:
We should avoid defining `debug` here, it's not really needed nor relevant.

Reply:
done

Reply:
Not, it's not ;-)

Is it possible to check tags elsewhere?
Why not defining a class method `check_format` that can be called externally ?
It may be useful for other uses.